.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -229,8 +229,12 @@ jobs:
           yq eval -i ".config-be-user-transformer.image.tag=\"$TAG_NAME\"" values.prod.yaml
           git add values.prod.yaml
 
+          cd ../../customer-objects/multi-tenant-us
+          yq eval -i ".spec.transformer.image.version=\"$TAG_NAME\"" hostedmtedmt.yaml
+          yq eval -i ".spec.user_transformer.image.version=\"$TAG_NAME\"" hostedmtedmt.yaml
+          git add hostedmtedmt.yaml
+
           git commit -m "chore: upgrade shared transformers to $TAG_NAME"
           git push -u origin shared-transformer-$TAG_NAME
 
           hub pull-request -m "chore: upgrade shared transformers to $TAG_NAME"
```

Comment:
Is hub available locally to github action runner instance?

Reply:
yes